docker/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of the IoT Agent for the Ultralight 2.0 protocol (IOTAUL) component
+#
+# IOTAUL is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+#
+# IOTAUL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with IOTAUL.
+# If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License
+# please contact with: [daniel.moranjimenez@telefonica.com]
+
+ARG NODE_VERSION=8.12.0-slim
+FROM node:${NODE_VERSION}
+ARG GITHUB_ACCOUNT=telefonicaid
+ARG GITHUB_REPOSITORY=iotagent-ul
+ARG DOWNLOAD_TYPE=latest
+
+ENV GITHUB_ACCOUNT=${GITHUB_ACCOUNT}
+ENV GITHUB_REPOSITORY=${GITHUB_REPOSITORY}
+ENV DOWNLOAD_TYPE=${DOWNLOAD_TYPE}
+
+MAINTAINER FIWARE IoTAgent Team. Telefónica I+D
+
+#
+# The following line retrieves the latest source code from GitHub.
+# 
+# To obtain the latest stable release run this Docker file with the parameters
+# --no-cache --build-arg DOWNLOAD_TYPE=stable
+#
+# Alternatively for local development, just copy this Dockerfile into file the
+# root of the repository and copy over your local source using : 
+#
+# COPY . /opt/iotaul/
+#
+RUN if [ ${DOWNLOAD_TYPE} = "latest" ] ; then RELEASE="master"; else RELEASE=$(curl -s https://api.github.com/repos/${GITHUB_ACCOUNT}/${GITHUB_REPOSITORY}/releases/latest | grep 'tag_name' | cut -d\" -f4); fi && \
+  	if [ ${DOWNLOAD_TYPE} = "latest" ] ; then echo "INFO: Building Latest Development"; else echo "INFO: Building Release: ${RELEASE}"; fi && \
+  	apt-get update && \
+  	apt-get install -y  --no-install-recommends unzip && \
+  	wget --no-check-certificate -O source.zip https://github.com/${GITHUB_ACCOUNT}/${GITHUB_REPOSITORY}/archive/${RELEASE}.zip && \
+	unzip source.zip && \
```

Comment:
For cleanness, source.zip should be removed after use.

Reply:
Fixed e290fb5